cpp/benchmarks/common/generate_input.cu

```diff
@@ -866,7 +866,7 @@ std::unique_ptr<cudf::table> create_random_table(std::vector<cudf::type_id> cons
                                                  unsigned seed)
 {
   auto const avg_row_bytes =
-    std::accumulate(dtype_ids.begin(), dtype_ids.end(), 0., [&](size_t sum, auto tid) {
+    std::accumulate(dtype_ids.begin(), dtype_ids.end(), 0., [&](double sum, auto tid) {
```

Comment:
```suggestion
    std::accumulate(dtype_ids.begin(), dtype_ids.end(), double{0}, [&](auto sum, auto tid) {
```

Reply:
This change actually has nothing to do with C++20, I just found the error and accidentally included the file in the commit. If folks don't mind, I'll merge it with the rest instead of having a separate PR.